<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopViewDemoTracerMainWindow</class>
 <widget class="QMainWindow" name="TopViewDemoTracerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopViewDemoTracerMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widgetForButtons" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <widget class="QPushButton" name="pushButtonNextGameStep">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>NextGameStep</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonKillBug">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>75</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Live On/Off</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFrameInd">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>10</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Bug #</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxBugNum">
       <property name="geometry">
        <rect>
         <x>151</x>
         <y>9</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPrintStat">
       <property name="geometry">
        <rect>
         <x>414</x>
         <y>10</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Print Statistics</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutMain">
      <item>
       <widget class="QListWidget" name="listWidgetTracks">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TrackerArenaForm" name="glWidgetTrace" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TrackerArenaForm</class>
   <extends>QWidget</extends>
   <header>TrackerArenaForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
